templates/field/field--entity-reference-revisions--paragraph.html.twig

```diff
@@ -1,13 +0,0 @@
-{#
-/**
- * @file
- * Theme override for the entity reference revisions field wich allowed to use paragraphs.
- *
- * @see ./templates/field/field--bare.html.twig
- */
-#}
-{% for item in items %}
-  <div class="ecl-u-mv-s">
-    {{ item.content }}
-  </div>
-{% endfor %}
```

Comment:
Why the decision to remove this? It looked quite convenient to have them applied to all paragraphs, present and future.

Reply:
The goal of this template is to add `<div class="ecl-u-mv-s">` foreach item.content, but it's the source of bug of this ticket.
we could keep this template without the div, it seems duplicate code, is it not?